owl/SOMA-ACT.owl

```diff
@@ -204,6 +308,53 @@ Also defined here are execution state regions, i.e. flags with which to label th
     
 
 
+    <!-- http://www.ease-crc.org/ont/SOMA-ACT.owl#AnsweringTask -->
+
+    <owl:Class rdf:about="http://www.ease-crc.org/ont/SOMA-ACT.owl#AnsweringTask">
```

Comment:
Shouldn't this task have a role for the answer message?

Reply:
Added!